agent/bench-scripts/pbench-uperf

```diff
@@ -182,10 +182,10 @@ function stop_server {
 }
 
 function install_uperf {
-	if check_install_rpm $benchmark_rpm $ver; then
-		debug_log "[$script_name]$benchmark_rpm is installed"
+	if check_install_rpm ${benchmark_rpm} ${ver}; then
+		debug_log "[$script_name] ${benchmark_rpm} is installed"
 	else
-		error_log "[$script_name]$benchmark_rpm installation failed, exiting"
+		error_log "[$script_name] ${benchmark_rpm} installation failed, exiting"
```

Comment:
We don't install any longer, so the message is wrong.

Reply:
Fixed.